pkg/security/security.go

```diff
@@ -69,7 +69,7 @@ var (
 	XDSTokenType = env.RegisterStringVar("XDS_TOKEN_TYPE", "Bearer",
 		"Token type in the Authorization header.").Get()
 
-	BearerTokenPrefix = XDSTokenType + " "
+	XDSBearerTokenPrefix = XDSTokenType + " "
```

Comment:
not sure why its changed to "XDS" when it applies to everything (CA)?

Reply:
BearerTokenPrefix is tied to XDSTokenType and only for XDS. In an offline discussion, Costin commented that  BearerTokenPrefix should be changed to XDSBearerTokenPrefix.

Reply:
Ok, the implementation does not match though. You changed ExtractBearerToken which is shard by CA.

In Istiod, the CA and XDS server have identical authentication configuration. This is intentional.

Reply:
Good point. However for external CA they are different.